dags/analyzer_helper/discord/discord_extract_raw_infos.py

```diff
@@ -0,0 +1,56 @@
+from datetime import datetime
+
+from analyzer_helper.discord.extract_raw_info_base import ExtractRawInfosBase
+from hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class DiscordExtractRawInfos(ExtractRawInfosBase):
+    def __init__(self, guild_id: str, platform_id: str):
+        """
+        Initializes the class with a specific guild and platform identifier.
+
+        Parameters
+        ----------
+        guild_id : str
+            The identifier for the guild.
+        platform_id : str
+            The identifier for the platform.
+        """
+        super().__init__(guild_id)
+        self.client = MongoSingleton.get_instance().client
+        self.guild_db = self.client[self.get_guild_id()]
+        self.platform_db = self.client[platform_id]
+        self.collection = self.guild_db["rawinfos"]
+        self.rawmemberactivities_collection = self.platform_db["rawmemberactivities"]
+
+    def extract(self, period: datetime, recompute: bool = False) -> list:
+        """
+        Extracts raw information data from the 'rawinfos' collection.
+
+        Parameters
+        ----------
+        period : datetime
+            The starting date from which data should be extracted.
+        recompute : bool, optional
+            If True, extracts all data from the collection. If False, extracts data
+            starting from the latest saved record's 'createdDate'.
+
+        Returns
+        -------
+        list
+            A list of documents from the 'rawinfos' collection.
+        """
+        data = []
+        if recompute:
+            data = list(self.collection.find({}))
+        else:
+            # Fetch the latest joined date from rawmemberactivities collection
+            latest_activity = self.rawmemberactivities_collection.find_one(sort=[("joinedAt", -1)])
```

Comment:
The rawmemberactivities collection doesn't have the `joinedAt` field. It has the `date` field. Please consider updating the sorting field to `date`, also please update the variable name `latest_joined_at` to something more appropriate here as we're extracting rawinfos but not guildmembers data.

Please have a close look at the rawmemberactivities data structure again: https://rndadocs.notion.site/Analyzer-Discord-Pipeline-5009512d82a243deb35b3388e17b29c5#:~:text=rawmemberactivities%20collection%20(transformed%20rawinfos)